bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -10,7 +10,7 @@ The primary thing type is the calendar.
 It is based on a single iCalendar file and implemented as bridge.
 There can be multiple things having different properties representing different calendars.
 
-Each calendar can have event filters which allow to get multiple events, maybe filtered by additional criteria. Time based filtering is done by each event's start.
+Each calendar can have event filters which allow to get multiple events, maybe filtered by additional criteria. Standard time-based filtering is done by each event's start, but it can also be configured to match other aspects.
```

Comment:
```suggestion
Each calendar can have event filters which allow to get multiple events, maybe filtered by additional criteria.
Standard time-based filtering is done by each event's start, but it can also be configured to match other aspects.
```